resources/source-types/sap_hana.yaml

```
@@ -0,0 +1,269 @@
+apiVersion: bindplane.observiq.com/v1beta
+kind: SourceType
+metadata:
+  name: hana
+  displayName: SAP Hana
+  icon: /icons/sources/sap.svg
+  #description: SAP Hana metrics and logs
+  description: SAP Hana logs
```

```suggestion
  description: Collect logs from SAP Hana 
```

I'm working on a updating our Stanza style guide for us to use with BindPlane OP so we have a standard here. For now, let's use the above format for descriptions